<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNodeWindow</class>
 <widget class="QDialog" name="AddNodeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Bahnschrift SemiCondensed&quot;;
background-color: rgb(197, 197, 147);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Категория</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="categoryBox">
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Красота и уход за собой</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лекарства и здоровье</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Коммуникации и связь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жизнь и развлечения</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Продукты</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Доход</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Одежда, товары для дома и прочее</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Сумма</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="sumEdit">
     <property name="maxLength">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0,0,0);
color: rgb(0, 0, 0);
border-width: 20px;
border-radius: 15px;
background-color: rgb(194, 194, 194);
font: 13pt &quot;Bahnschrift SemiCondensed&quot;;</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0,0,0);
color: rgb(0, 0, 0);
border-width: 20px;
border-radius: 15px;
background-color: rgb(194, 194, 194);
font: 13pt &quot;Bahnschrift SemiCondensed&quot;;</string>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="currentRadio">
        <property name="text">
         <string>Сегодня</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="otherRadio">
        <property name="text">
         <string>Другое</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
